modular_skyrat/modules/horrorform/code/modules/mob/hostile/true_changeling.dm

```diff
@@ -0,0 +1,212 @@
+	button_icon = 'modular_skyrat/modules/horrorform/icons/mob/actions/actions_changeling.dmi'
+	background_icon_state = "bg_changeling"
+	button_icon_state = "change_to_human"
+
+/datum/action/innate/turn_to_human/Trigger()
+	var/mob/living/simple_animal/hostile/true_changeling/C = owner
+	if(!C.stored_changeling)
+		to_chat(C,"<span class='warning'>We do not have a form other than this!</span>")
+		return 0
+	if(C.stored_changeling.stat == DEAD)
+		to_chat(C,"<span class='warning'>Our human form is dead!</span>")
+		return 0
+	if(world.time - C.transformed_time < TRUE_CHANGELING_REFORM_THRESHOLD)
+		var/timeleft = (C.transformed_time + TRUE_CHANGELING_REFORM_THRESHOLD) - world.time
+		to_chat(C,"<span class='warning'>We are still unable to change back at will! We need to wait [round(timeleft/600)+1] minutes.</span>")
+		return 0
+	C.visible_message("<span class='warning'>[C] suddenly crunches and twists into a smaller form!</span>", \
+						"<span class='danger'>We return to our lesser form.</span>")
+	C.stored_changeling.loc = get_turf(C)
+	C.mind.transfer_to(C.stored_changeling)
+	C.stored_changeling.Stun(2 SECONDS)
+	C.stored_changeling.status_flags &= ~GODMODE
+	qdel(C)
+	return 1
+
+/datum/action/innate/devour
+	name = "Devour"
+	desc = "We tear into the innards of a human. After some time, they will be significantly damaged and our health partially restored."
+	icon_icon = 'modular_skyrat/modules/horrorform/icons/mob/actions/actions_changeling.dmi'
+	background_icon_state = "bg_changeling"
+	button_icon_state = "devour"
+
+/datum/action/innate/devour/Trigger()
+	var/mob/living/simple_animal/hostile/true_changeling/T = owner
+	if(T.devouring)
+		T << "<span class='warning'>We are already feasting on a human!</span>"
+		return 0
+	var/list/potential_targets = list()
+	for(var/mob/living/carbon/human/H in range(1, usr))
+		if(H == T.stored_changeling || (H.mind && H.mind.has_antag_datum(/datum/antagonist/changeling))) //You can't eat changelings in human form
+			continue
+		potential_targets.Add(H)
+	if(!potential_targets.len)
+		T << "<span class='warning'>There are no humans nearby!</span>"
+		return 0
+	var/mob/living/carbon/human/lunch
+	if(potential_targets.len == 1)
+		lunch = potential_targets[1]
+	else
+		lunch = input(T, "Choose a human to devour.", "Lunch") as null|anything in potential_targets
+	if(!lunch && !ishuman(lunch))
+		return 0
+	T.devouring = TRUE
+	T.visible_message("<span class='warning'>[T] begins ripping apart and feasting on [lunch]!</span>", \
+					"<span class='danger'>We begin to feast upon [lunch]...</span>")
+	if(!do_mob(usr, 50, target = lunch))
+		T.devouring = FALSE
+		return 0
+	T.devouring = FALSE
+	if(lunch.getBruteLoss() + lunch.getFireLoss() >= 200) //Overall physical damage, basically
+		T.visible_message("<span class='warning'>[lunch] is completely devoured by [T]!</span>", \
+						"<span class='danger'>You completely devour [lunch]!</span>")
+		lunch.gib()
```

Comment:
Consider removing this part, permanent round removal bad

Reply:
This is intended as a balance feature. And generally speaking providing the head remains they shouldn't be permanently round removed.